scripts/run_multi_model.py

```
@@ -0,0 +1,372 @@
+            capture_output=True, text=True,
+            cwd=os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        )
+        end = datetime.now(timezone.utc)
+        duration = (end - start).total_seconds()
+
+    # Parse pytest output for pass/fail counts
+    output = result.stdout + result.stderr
+    
+    import xml.etree.ElementTree as ET
+    try:
+        tree = ET.parse(junit_xml_path)
```

Ruff (0.15.9)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)